docs/pages/preview/upcoming-releases.mdx

```diff
@@ -10,21 +10,34 @@ The teleport team delivers a new major release roughly every 3 months.
 
 ### Teleport 9.1
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for the CLI and the Web UI.
 
 | Version     | Date             |
 |-------------|------------------|
-| Release     | Early April 2022 |
+| Release     | April 18th 2022  |
 
 ### Teleport 9.2
 
-Teleport 9.2 will include Teleport Terminal, a native application that can be
-used to access Teleport Servers and Databases.
+Teleport 9.2 will add Database and Kubernetes Access support for Machine ID and
+clickable label filters for the Web UI.
+
+In addition, Teleport 9.2 will include Teleport Terminal, a native application
+that can be used to access Teleport Nodes and Databases.
+
+| Version     | Date            |
+|-------------|-----------------|
+| Release     | Early May 2022  |
```

Comment:
I thought we were planning on shipping Machine ID X.509 support and Teleport Terminal preview in 9.1 (and delaying that release to 4/29).